jobsdb/jobsdb.go

```diff
@@ -3215,7 +3215,7 @@ func (jd *HandleT) getBackUpQuery(backupDSRange *dataSetRangeT, isJobStatusTable
 	if jd.BackupSettings.FailedOnly {
 		// check failed and aborted state, order the output based on destination, job_id, exec_time
 		stmt = fmt.Sprintf(
-			`SELECT 
+			`SELECT
```

Comment:
Not using [gofumpt](https://github.com/mvdan/gofumpt)?

Reply:
Tried configuring using gofumpt with gopls too. It still removes the whitesapce as done in the commit